autoload/neomake.vim

```diff
@@ -528,17 +530,16 @@ function! neomake#GetEnabledMakers(...) abort
     return enabled_makers
 endfunction
 
-function! s:HandleLoclistQflistDisplay(file_mode) abort
-    let open_val = get(g:, 'neomake_open_list')
-    if open_val
+function! s:HandleLoclistQflistDisplay(open_val, file_mode) abort
```

Comment:
I hate vimscript and not very excited about writing the tests :p 
Which test would you recommand to use as a base ? or should I extend an existing one ?